plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -99,6 +101,23 @@ protected DistributedQueryRunner createQueryRunner()
         }
     }
 
+    @Test
+    public void testCreateTableAsSelect()
+    {
+        assertFileSystemAccesses(
+                "CREATE TABLE test_create_as_select AS SELECT 1 col_name",
+                ImmutableMultiset.<FileOperation>builder()
+                        .addCopies(new FileOperation(STARBURST_EXTENDED_STATS_JSON, "extendeded_stats.json", INPUT_FILE_NEW_STREAM), 1)
```

Comment:
pre-existing `extendeded_stats.json` is gramatically incorrect. 
In case it would get changed, this would likely be in a different PR

Reply:
It's not changing in this PR, it's still the same spelling as before

Reply:
it is has been here for a while and i doubt we can fix it now :D